start-chrome-for-playr.sh

```diff
@@ -35,7 +39,88 @@ no_nagging_options="--disable-features=SameSiteByDefaultCookies,CookiesWithoutSa
 # The path to the page that will check internet connection
 # before loading the actual signage channel
 # NOTE: check the location of the player_loader.html in the following line
-playr_loader_file="$HOME/playr-loader/playr_loader.html"
+execution_path=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+playr_loader_file="${execution_path}/playr_loader.html"
+
+# Add some terminal colors
+COLOR_OFF='\033[0m'       # Text reset
+COLOR_RED='\033[0;31m'    # Red
+COLOR_YELLOW='\033[0;33m' # Yellow
+COLOR_BLUE='\033[0;34m'   # Blue
+COLOR_GREEN='\033[0;32m'  # Green
+
+##########################################################################
+#							   METHODS     								 #
+##########################################################################
+
+# Use this to write error messages to the terminal
+log_error() {
+	echo -e "[ERROR] - $(date +%F-%T) - $COLOR_RED${1}$COLOR_OFF"
+}
+
+# parse mac address from the ip link command
+# ignore loopback devices and focus on link/ether
+# extract the mac and select the first one based on activation order in kernel
+parse_mac_from_ip_link() {
+	echo $(ip link | grep -E "link/ether" | grep -o -E '([[:xdigit:]]{2}:){5}[[:xdigit:]]{2}' | head -1)
+}
+
+# get first mac of the network hardware
+# order defined by kernel activation order
+get_first_hardware_mac() {
+	if ! which ip >/dev/null; then
+		echo "ip not installed on this system, please install ip"
```

Comment:
laten we hier log_warning ipv echo gebruiken

Reply:
We hebben hier gekozen om de error als string te returnen en pas op het einde van execute log_error te gebruiken.
Aangezien get_first_hardware_mac aangeroepen wordt door get_system_uuid en we de reden van de error willen behouden.